includes/definitions/mrv-optidriver.yaml

```diff
@@ -0,0 +1,10 @@
+os: mrv-optidriver
+group: mrv
+text: 'MRV OptiDriver'
+type: network
+icon: mrv
+discovery:
+    - sysObjectId:
+       - .1.3.6.1.4.1.629.200.1.1.1.
```

Comment:
The 1.1.1 is what's showing in the full snmpget of sysObjectID.

```
.1.3.6.1.2.1.1.2.0 = OID: .1.3.6.1.4.1.629.200.1.1.1
SNMPv2-MIB::sysObjectID.0 = OID: NBS-CMMC-MIB::nbsCmmcObjects.1.1
```

Still want it shorted?